code/__HELPERS/threat.dm

```diff
@@ -0,0 +1,47 @@
+/proc/get_threat_level_string(shown_threat, fake_greenshift)
```

Comment:
This isn't an improvement if this proc is used just once in 1 file, all this does is add proc overhead & an extra file to compile.

Put back the original code

Reply:
I disagree, it's more self-documenting code.

Reply:
Proc overhead isn't a concern unless something is called thousands of times a second

A proc like this is fine for organization's sake, it makes reading the file less cumbersome because it's like 100 lines

Reply:
It's not that proc overhead is a big concern but more like why add it when you don't have to.

> I disagree, it's more self-documenting code.

Fair enough. Remember to autodoc the proc then with params

Reply:
okay, you code it. My solution is better than the current, don't you think?

Reply:
I agree that I'm fine with this being its own proc but personally I would have located it in the same file as the only place it is called, not really a big deal either way though.

Reply:
why is it in the global namespace